spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -191,22 +153,22 @@ def create_workspace_csv(workspaces, client_id_column)
     @data_product_id = "DATA_PRODUCT_#{@suffix}"
 
     @test_context = {
-        release_table_name: @release_table_name,
-        workspace_table_name: @workspace_table_name,
-        config: @config,
-        ads_client: @ads_client,
-        jdbc_url: @ads.data['connectionUrl'],
-        development_pid: @project.obj_id,
-        segments: segments.to_json,
-        segments_filter: segments_filter.to_json,
-        ads_output_stage_uri: production_output_stage_uri,
-        data_product: @data_product_id,
-        input_source_type: 's3',
-        s3_bucket: bucket_name,
-        s3_endpoint: s3_endpoint,
```

Comment:
error: Identical code found in shared_contexts_for_lcm.rb:197, vertica_end_to_end_spec.rb:82 (mass = 168) (flay)